app.py

```diff
@@ -77,4 +77,7 @@ def update_movie(movie_id: int):
 @app.post('/movies/<int:movie_id>/delete')
 def delete_movie(movie_id: int):
     # TODO: Feature 6
-    pass
+    # Taking in movie_id parameter and then calling delete_movie method on the movie_repository to delete the corresponding movie
+    movie_repository.delete_movie(movie_id)
+    # Redirecting back to the movie page
+    return redirect('/movies')
```

Comment:
Correct use of delete_movie method to delete a movie from the dictionary and correct redirect to the /movies to render the table after a delete.

Reply:
Yeah not much to do with deleting the movie, looks good I think as well